### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by SeanZoR**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Getting Started
+If you're new to contributing to open source projects, we recommend checking out the [Open Source Guides](https://opensource.guide/) for an introduction to the process.
+
+To get started with contributing to Lava, familiarize yourself with the project by reading the project's [litepaper](https://lavanet.xyz/), Repo's README and documentation, trying out the project, and joining the project's Discord channels to ask questions and learn from other contributors.
+
+Support is detailed in [SUPPORT.md](SUPPORT.md)
+
+## Coding Guidelines
+Please follow these guidelines when contributing code to Lava:
+
```
Comment (workingrush) [inline CONTRIBUTING.md:27]: Consider adding a section or paragraph describing how to set up a local development environment, and how contributors can run tests locally

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+* Include tests for any new code
+* Follow the project's coding style (e.g. indentation, naming conventions)
```
Comment (workingrush) [inline CONTRIBUTING.md:29]: Is there a link where the coding style is defined?

File: CONTRIBUTING.md (hunk 1 above)
Comment (SeanZoR) [reply CONTRIBUTING.md:27]: Thanks @workingrush! for starters, this general contributing guidelines will be good enough to start. 
We'll probably add ad-hoc contributing guidelines for specific repo's.

File: CONTRIBUTING.md (hunk 2 above)
Comment (SeanZoR) [reply CONTRIBUTING.md]: No coding style, yet. (@omerlavanet)
I've marked that part as TBD.

**Pull request merged**